print/resources/style.css

```diff
@@ -573,6 +577,11 @@ body {
     overflow: hidden;
   }
 
+  figure.pattern-only {
+      border: none !important;
+      height: 100% !important;
+  }
+
```

Comment:
`height: 100%` doesn't do anything, but we know the page size and can go for mm values. We good choose e.g. `265mm` here. For both values we do not need `!important`